introduction.md

```diff
@@ -0,0 +1,2 @@
+Hi, I am 44Nico44 and I am happy to enter in the dojo.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.